x-pack/solutions/observability/plugins/streams_app/public/components/stream_detail_routing/preview_panel.tsx

```diff
@@ -176,7 +176,7 @@ export function PreviewPanel({
   return (
     <>
       <EuiFlexItem grow={false}>
-        <EuiFlexGroup alignItems="center">
+        <EuiFlexGroup alignItems="flexStart">
```

Comment:
What is this change for? The layout if this page is quite delicate giving the table sizing, is this need to fix something?